converter/src/test/java/gov/cms/qpp/conversion/validate/CheckerTest.java

```diff
@@ -169,6 +169,46 @@ void testChildrenMaximumSuccess() {
 				.that(details).isEmpty();
 	}
 
+	@Test
+	void testChildExactFailure() {
+		Node meepNode = new Node();
+		meepNode.addChildNodes(new Node(TemplateId.PLACEHOLDER),
+			new Node(TemplateId.ACI_AGGREGATE_COUNT),
+			new Node(TemplateId.PLACEHOLDER));
+
+		Checker checker = Checker.check(meepNode, details);
+		checker.childExact(ERROR_MESSAGE, 2, TemplateId.PLACEHOLDER, TemplateId.ACI_AGGREGATE_COUNT);
+
+		assertWithMessage("An error exists")
+			.that(details).hasSize(1);
```

Comment:
We should test that we get the expected error and not just the correct number of errors.